base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2951,7 +2899,25 @@ fn run_migrations(db: &LMDBDatabase) -> Result<(), ChainStorageError> {
             txn.commit()?;
             info!(target: LOG_TARGET, "added RX vm key 91ef83186cefaa646dc4c6e950e68e4debab52b4f4a9b7f465891e91fe5f6ce4");
         }
+        if migrate_from_version == 4 {
+            let txn = db.write_transaction()?;
+            info!(target: LOG_TARGET, "Clearing bad blocks list missed in migration v3");
+            clear_bad_blocks = true;
+        }
     }
+
+    // NOTE: If you are adding a new migration, consider deleting bad blocks, even if there are none at the time.
+    // (There might be new ones by the time this is released.)
+    // If you change a validation rule, a bad block will still be banned even if the rule has changed. So in most
+    // cases you will want to clear it.
+    if clear_bad_blocks {
+        let txn = db.write_transaction()?;
+        info!(target: LOG_TARGET, "Clearing bad blocks list");
```

Comment:
We have the info log just before this one
```suggestion
        debug!(target: LOG_TARGET, "Clearing bad blocks list");
```